assets/js/script.js

```diff
@@ -0,0 +1,353 @@
+
+let startButton = document.getElementById("startBtn");
+let questionContainerEl = document.getElementById("questionCard");
+let questionEl = document.getElementById("question");
+let answerBtnsEl = document.getElementById("answerBtns");
+var currentQuestion = 0;
```

Comment:
Use let instead of var here. 

https://stackoverflow.com/questions/762011/what-is-the-difference-between-let-and-var